SFDigitalServices/sf-dahlia-web
# DAH-1572 add flag, update tests

Jira: https://sfgovdt.jira.com/browse/DAH-1572

Changes
- In the cache service, check if variable, 'CACHE_LISTING_IMAGES' in .env file is set to true before running the image cache service

Testing
 - In your .env file, create a new variable: `CACHE_LISTING_IMAGES='true'`
 - Run the caching service by running `./bin/rake cache:prefetch_daily` in the terminal or the vscode debugger
 - Ensure that when the env variable is set to true, the image service is called. When it is set to false, it should not be called

Next Steps
- In either the QA or Full Env, deploy these changes and set the CACHE_LISTING_IMAGES='false'

Follow instructions from Jira ticket:
- Check the url for a few listing images by right clicking on an image and select “open in new tab” (at least in chrome). 

With the cache - most images should have a url that looks something like this -  ‘https://d31h23p2gsu1kr.cloudfront.net/’  (note the [cloudfront.net](http://cloudfront.net/) portion)

Without the cache - most images should have a url that looks something like this - '[https://sfmohcd.org/sites/default/files/Documents/MOH/Housing Listing Photos/Cameo Apts 8.30.22.jpg](https://sfmohcd.org/sites/default/files/Documents/MOH/Housing%20Listing%20Photos/Cameo%20Apts%208.30.22.jpg)' (note the [sfmohcd.org](http://sfmodcd.org/) portion)
